src/components/Documentation/MobileLeftNav.js

```diff
@@ -0,0 +1,138 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+import { docType } from "constants/docType";
+import { MEDIA_QUERIES, PALETTE } from "constants/styles";
+
+import { CloseIcon, Menu } from "basics/Icons";
+
+import { BasicButton } from "basics/Buttons";
+import { Link } from "basics/Links";
+
+import {
+  BetaBadge,
+  NavDivider,
+  NavLogo,
+} from "components/Navigation/SharedStyles";
+import { LeftNav } from "./LeftNav";
+
+const El = styled.div`
+  position: fixed;
+  z-index: 10;
+  @media (${MEDIA_QUERIES.gtTablet}) {
+    display: none;
+  }
+`;
+const NavIconSectionEl = styled.div`
+  display: flex;
+  align-items: center;
+`;
+const NavBarEl = styled.div`
+  display: flex;
+  width: 200vw;
+  align-items: center;
+  background-color: ${PALETTE.white};
+  border-bottom: 1px solid ${PALETTE.white60};
+`;
+const NavSectionEl = styled.div`
+  flex-basis: 100vw;
+  padding: 0 1rem;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+`;
+const CloseIconEl = styled(CloseIcon)`
+  width: 1rem;
+  height: 1rem;
+`;
+const MenuButtonEl = styled(BasicButton)`
+  padding: 1rem;
+  margin: 0 -1rem;
+`;
+const OffscreenContainerEl = styled.div`
+  width: 100vw;
+  transform: translate(${(props) => (props.isOpen ? "0" : "-100%")});
+  transition: transform 0.2s;
```

Comment:
Forgot to update this to use a constant after hardcoding initially, will follow up